lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -496,6 +496,39 @@ static int process_cmd(const tunnel_comand *cmd, command_cb cb, void *ctx) {
             return 0;
         }
 
+        case TunnelCommand_RemoveIdentity: {
+            tunnel_delete_identity delete_id = {0};
+            if (cmd->data == NULL || parse_tunnel_delete_identity(&delete_id, cmd->data, strlen(cmd->data)) != 0) {
+                result.success = false;
+                result.error = "invalid command";
+                free_tunnel_delete_identity(&delete_id);
+                break;
+            }
+            struct ziti_instance_s *inst = model_map_get(&instances, delete_id.identifier);
+
+            if (inst == NULL) {
+                result.error = "ziti context not found";
+                result.success = false;
+                free_tunnel_delete_identity(&delete_id);
+                break;
+            }
+            if (inst) {
+                if (inst->ztx && ziti_get_identity(inst->ztx)) {
+                    disconnect_identity(inst->ztx, CMD_CTX.tunnel_ctx);
+                }
+                model_map_remove(&instances, delete_id.identifier);
+                result.success = true;
+                result.data = tunnel_comand_to_json(cmd, MODEL_JSON_COMPACT, NULL);
+            } else {
+                result.success = false;
+                result.error = calloc(1, strlen(delete_id.identifier) + 35);
```

Comment:
when does `error` get processed? will it be freed?

Reply:
result will be converted to json in the callback function (on_command_resp in ziti-edge-tunnel.c) and the json message will be sent to the IPC queue

Reply:
I will remove this calloc function call